samples/simple-chat/csharp/FunctionApp/Functions.cs

```diff
@@ -59,6 +59,16 @@ public static class Functions
             await serviceHubContext.Clients.All.SendAsync("newMessage", message);
         }
 
+        // curl -X POST -H "Content-type: application/json" -d '{"HubName": "simplechat", "Data": {"Target": "broadcast2", "Arguments": ["hello"]}}' localhost:7071/runtime/webhooks/signalr/simplechat
+        [FunctionName("broadcast2")]
+        public static async Task BroadcastWithSignalRTrigger(
+            [SignalRTrigger(HubName = "simplechat")]InvocationContext context)
+        {
+            var message = (string)context.Data.Arguments[0];
+            var serviceHubContext = await StaticServiceHubContextStore.Get().GetAsync("simplechat");
```

Comment:
>StaticServiceHubContextStore [](start = 42, length = 28)

`StaticServiceHubContextStore` is only available for C#. 
For general languages, is it possible adding a new property in InvocationContext to tell what kind of methods (broadcast, echo, etc.) and send the message behind?

Reply:
Can we get rid of this API completely?

Reply:
@davidfowl  `InvocationContext` need to be available in various languages, it can only contain properties without methods. A solution is to build some SDKs in popular languages, especially `js` . Customers can just import these SDK packages instead of using output binding. Thus, they can use the similar pattern `..Clients.All.SendAsync(...)`. But it's another topic about how making output direction much easier to use. I think it's hard to integrate with the input Trigger.